validator/validator-main.protoascii

```diff
@@ -3303,6 +3303,14 @@ attr_lists: {
   }
 }
 
+# Video Interface attributes. These attributes can be attached to any
+# element that implements the video interface
+attr_lists: {
+  name: "implements-video-interface"
+  attrs: { name: "autoplay" }
+  attrs: { name: "dock" }
```

Comment:
@honeybadgerdontcare @lannka we don't want `dock` to be available in A4A, is there a way to do that and still keep a common `attr_lists`?